### PR TITLE
Windows peeps

### PR DESCRIPTION
**Pull request by CarlyLanglois**
adds and appendix page for windows help on asp apps

### CODE REVIEW

File: src/appendices/vs-troubleshooting.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.. _vs-troubleshooting:
+
+Visual Studio Operational Tips
+==============================
+
+For Windows users, opening and running some of the ASP.NET applications we provide in this book may require some extra steps. 
+Here are some tips to help you troubleshoot. You may find some (or none!) of them relevant to you.
+
+- Weather you clone a project directly onto your computer and open it in Visual Studio, or choose to find a project to clone from the Visual Studio
```
Comment (gildedgardenia) [inline src/appendices/vs-troubleshooting.rst]: "Whether"

File: src/appendices/vs-troubleshooting.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  *Solution Explorer* window, go to *File > Start Window*. From the *Start Window*, find the ``.sln`` file for your project and open that directly 
+  (instead of any parent directory).
+
+   .. figure:: figures/vs-start-window.png
+      :alt: Image showing the project items available to be opened in Visual Studio.
+
+      The project items available to be opened in Visual Studio.
+
+- **BEFORE** you attempt to run or debug your program, we recommend navigating to *Debug > <ProjectName> Debug Properties*. This will open a 
+  window where you can change some application seetings. Scroll down to the bottom of the *Debug* view and under *Web Server Settings*, make sure
```
Comment (gildedgardenia) [inline src/appendices/vs-troubleshooting.rst]: "settings"

**Review by gildedgardenia — APPROVED** [review]
Looks great! I just found a few small typos and the tips and tricks are very helpful!

File: src/appendices/vs-troubleshooting.rst (hunk 1 above)
Comment (CarlyLanglois) [reply src/appendices/vs-troubleshooting.rst]: lol

**Pull request merged**